Eastern-Research-Group/expert-query
# Feature/208 link to userguide

## Related Issues:
* https://jira.epa.gov/browse/EQ-208

## Main Changes:
* Added a placeholder for the user's guide.

## Steps To Test:
1. Verify the user's guide button at the top of the page works.
2. Verify the national downloads page still works
3. Navigate to http://localhost:9090/api/getFile/path/config/services.json
4. Verify the file is opened in the browser
